src/field_theory/krull_topology.lean

```diff
@@ -187,3 +192,94 @@ group_filter_basis.topology (gal_group_basis K L)
 instance (K L : Type*) [field K] [field L] [algebra K L] :
   topological_group (L ≃ₐ[K] L) :=
 group_filter_basis.is_topological_group (gal_group_basis K L)
+
+section krull_t2
+
+
+/-- If `f,g : L → L` are distinct `K`-algebra equivalences, then there is some `x ∈ L` with
+  `f(x) ≠ g(x)`. -/
+lemma diff_equivs_have_diff_values {K L : Type*} [field K] [field L] [algebra K L]
+{f g : L ≃ₐ[K] L} (h : f ≠ g) :
+∃ (x : L), f x ≠ g x :=
+begin
+  contrapose! h,
+  exact alg_equiv.ext h,
+end
```

Comment:
This probably belongs in the `fun_like` namespace. Maybe @Vierkantor knows if we already have it somewhere.

Reply:
Assuming we don't already have it, what should I do about it? Should I move it to a different file within this same PR?

Reply:
Indeed, this would be a good addition to `data/fun_like/basic.lean:170`:
```lean
lemma exists_ne {f g : F} (h : f ≠ g) : ∃ x, f x ≠ g x :=
not_forall.mp $ mt (fun_like.ext _ _) h
```
The closest equivalent I know is `fun_like.coe_injective.ne`, but that doesn't do the extra `ext` step.

Reply:
Note there is now a zulip thread about this lemma

Reply:
Link: https://leanprover.zulipchat.com/#narrow/stream/217875-Is-there.20code.20for.20X.3F/topic/Functions.20are.20unequal.20if.20they.20disagree.20somewhere